static/css/style.css

```diff
@@ -1194,7 +1194,7 @@ pre > code {
           align-items: center;
   border-radius: 3px;
   background: #f5f5f5;
-  max-width: 550px;
+  max-width: 650px;
```

Comment:
why the change?

Reply:
because there wasn't room for the two check boxes...

Reply:
SG.